src/routes/committee/rehearsals/[rehearsalId]/availability/+page.ts

```diff
@@ -17,16 +17,18 @@ export const load: PageLoad = async ({ params }) => {
 	)
 	const allAvailability = (await getDocs(availabilityQuery)).docs
 
-	const userAvailabilities: { name: string; availability: availability | undefined }[] = users.map(
-		(user) => {
+	const userAvailabilities: { name: string; availability: availability | undefined }[] = users
+		.filter(
+			(user) => user.id !== 'Om9orRRBrIkeXhHTHELj' // Filter out admin account as it is not a user
```

Comment:
misschien env var, of op zn minst op de name filteren en niet id, dat is zo random

Reply:
Updated, added variable to users types file